samples/suit/ab/README.rst

```diff
@@ -32,30 +33,279 @@ To perform a SUIT update over UART using the SMP protocol, install SUIT commands
+
+The suit-install sequence
+-------------------------
+
+The logic inside the suit-install is very similar to the logic inside the suit-candidate-verification sequence.
+The main differences are:
+
+* The suit-install sequence from the local manifests is executed.
+  The main puprose of them is to check that the in-place update, performed by the application was successful.
+  If it is, the sequence completes and the local manifests are updated inside the SUIT storage area.
+* The logic updates the ``CONFIRM_SET_A`` or ``CONFIRM_SET_B`` variables before executing the suit-install sequence of the local manifests.
+* The logic updates the ``BOOT_PREFERENCE`` variable after executing the suit-install sequence of the local manifests.
+  This operation effectively schedules the verification of the newly installed images and manifests after a reboot.
+* If the Nordic manifest is present, the logic schedules update of the Nordic firmware binaries stored inside the temporary DFU cache partition.
+
+.. figure:: img/ab_root_install.png
+
+   :alt: The suit-install sequence of the root manifest
+
+Once the suit-install sequences are completed successfully, the root manifest is updated inside the SUIT storage and the SDFW reboots the device.
+After the reboot, the SDFW will proceed with the boot path, executing the suit-validate sequence of the new root manifest.
+
+The suit-validate sequence
+--------------------------
+
+The suit-validate sequence of the root manifest is used to select the valid set of manifests to boot from.
+The result of this check is written into the ``BOOT_STATUS`` manifest variable.
+The following activity diagram presents the logic, implemented inside the suit-validate sequence.
+
+.. figure:: img/ab_root_validate.png
+
+   :alt: The suit-validate sequence of the root manifest
+
+The suit-invoke sequence
+------------------------
+
+The suit-invoke sequence of the root manifest reads the ``BOOT_STATUS`` manifest variable value and act upon it.
+If the application has been just updated, which is indicated by the ``CONFIRM_SET_A`` or ``CONFIRM_SET_B`` manifest variable set to ``JUST_INSTALLED``,
+and correctly validated by the suit-validate sequence, the confirmation variable is reset to ``NOT_CONFIRMED``.
+It is now the applications responsibility to set this variable to ``CONFIRMED``.
+Otherwise this variant will be considered as degraded on the next boot attempt.
+The following activity diagram presents the logic, implemented inside the suit-invoke sequence.
+
+.. figure:: img/ab_root_invoke.png
+
+   :alt: The suit-invoke sequence of the root manifest
+
+It is worth noticing that apart from regular application/radio/root manifest, the application recovery manifest is also defined.
+In the A/B sample there is no separate recovery binary though.
+The recovery manifest is very similar to the root manifest instead, so in case of the root manifest damage, the recovery manifest serves as a backup logic.
```

Comment:
You can also add that the recovery manifest ignores the boot preference and istead simply tries to boot first the variant A and then the variant B and always boots them in "degraded" mode,